### PR TITLE
Fix bug in edit applicant

### PR DESCRIPTION
**Pull request by e0543517**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/applicant/EditApplicant.java [hunk 1]
```diff
@@ -74,8 +73,9 @@
             + " is the same as the current details of the Applicant";
     public static final String MESSAGE_DATE_APPLIED_LATER_THAN_INTERVIEW_DATE = "The date applied by this applicant "
             + "is later than the interview date of this applicant.";
-
-
+    public static final String MESSAGE_DUPLICATE_EMAIL = "The edited email of the applicant already exists.";
+    public static final String MESSAGE_DUPLICATE_NRIC = "The edited Nric of the applicant already exists.";
+    public static final String MESSAGE_DUPLICATE_PHONE = "The edited phone number of the applicant already exists.";
```
Comment (Justinnje) [inline src/main/java/seedu/address/logic/commands/applicant/EditApplicant.java:78]: Mayb can change email to Email, Nric to NRIC, phone number to Phone number to emphasize that word

**Review by Justinnje — COMMENTED** [review]
LGTM. Just one small change

File: src/main/java/seedu/address/logic/commands/applicant/EditApplicant.java (hunk 1 above)
Comment (e0543517) [reply src/main/java/seedu/address/logic/commands/applicant/EditApplicant.java]: Changed

**Review by Justinnje — APPROVED** [review]
LGTM

**Pull request merged**